resources/views/listing/partials/filter/swatch.blade.php

```diff
@@ -1,38 +1,37 @@
-<swatch-filter v-else-if="filter.text_swatch || filter.visual_swatch" v-slot="slotProps">
```

Comment:
Alleen swatch filter is weg gehaald en de `v-else-if="filter.text_swatch || filter.visual_swatch"` is verplaatst naar multi-list